hadoop-project/pom.xml

```diff
@@ -2152,6 +2155,11 @@
         <artifactId>jersey-media-json-jettison</artifactId>
         <version>${jersey2.version}</version>
       </dependency>
+      <dependency>
```

Comment:
Since we're globally dependency managing jaxb-runtime, I think that we don't need all the excludes in this patch.
They should all be dependency managed to 2.3.9 by maven, and the dependency convergence check should pass.

Reply:
I think that the reported error was for the earlier patch without the global dependency management.